doc/maintaining/installing/install-from-source.rst

```diff
@@ -80,6 +83,19 @@ a. Create a Python `virtual environment <http://www.virtualenv.org>`_
        sudo chown \`whoami\` |virtualenv|
        virtualenv --no-site-packages |virtualenv|
        |activate|
+    
+.. note::
+
+    If you are using ``python3`` and if you are having problems installing the
```

Comment:
Can we reword this section to something like:

```
.. note::
    If your system uses Python 3 by default (eg Ubuntu 18.04) make sure to create the virtualenv using the Python 2.7 executable with the ``--python`` option:

      .. parsed-literal::
         virtualenv --python=/usr/bin/python2.7 --no-site-packages |virtualenv|

```

Reply:
I have to admit I had problems paraphrasing this section. This is much better, you just wrote everything I was thinking exactly like it should be. Thanks!